### PR TITLE
WT-3447 Fix python test so it will loop checking for WiredTigerTables

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

**Comment by daveh86** [comment]
retest this please

**Comment by daveh86** [comment]
@sulabhM can you review this?

File: test/suite/test_stat_log02.py [hunk 1]
```diff
@@ -96,16 +96,21 @@ def check_file_is_json(self, file_name):
...
-        has_tables = False
-        for line in f:
-            data = json.loads(line)
-            if "wiredTigerTables" in data:
-                if "file:foo.wt" in data["wiredTigerTables"]:
-                    has_tables = True
+        # We wait for another 30 sleeps here to avoid erroring in the case where
+        # the stat log has only made the first pass and not yet printed the
+        # individual table stats.
+        num_sleeps = 0
```
Comment (sulabhM) [inline test/suite/test_stat_log02.py]: maybe you mean number_sleeps = 0

**Comment by daveh86** [comment]
retest this please

**Comment by daveh86** [comment]
@agorrod can you have a look at this?

**Comment by agorrod** [comment]
lgtm

**Pull request merged**